packages/pieces/youtube/src/lib/common/props.ts

```diff
@@ -0,0 +1,6 @@
+import { Property } from "@activepieces/framework";
+
+export const youtubeFeedUrl = Property.ShortText({
+    displayName: 'YouTube Feed URL',
```

Comment:
I would change the name to be something like "YouTube Video RSS Feed" as this could cause some confusion and people might just this URL 'https://www.youtube.com/<channel>/videos' instead of the RSS feed

Reply:
I agree with you @ShayPunter , but I would rename it to CHANNEL_ID and append it to the url inside the piece to avoid confusion.